app/src/main/java/de/danoeh/antennapod/fragment/preferences/GpodderPreferencesFragment.java

```diff
@@ -115,7 +115,7 @@ private void updateGpodnetPreferenceScreen() {
             String format = getActivity().getString(R.string.pref_gpodnet_login_status);
             String summary = String.format(format, GpodnetPreferences.getUsername(),
                     GpodnetPreferences.getDeviceID());
-            findPreference(PREF_GPODNET_LOGOUT).setSummary(Html.fromHtml(summary));
+            findPreference(PREF_GPODNET_LOGOUT).setSummary(HtmlCompat.fromHtml(summary, HtmlCompat.FROM_HTML_MODE_LEGACY));
```

Comment:
This line is too long, now. That's why CircleCI complains :)

Reply:
Ups. Let's see if this is short enough. (I could not reproduce the error by running the `checkstyle` task locally, though.)

Reply:
Yeah, the checkstyle gradle task only checks the things that AntennaPod already fulfills. We use diff-checkstyle on CI to gradually enforce more rules while still not doing a huge "reformatted everything" PR that messes up merges and git history. Diff-checkstyle uses the [checkstyle-new-code.xml](https://github.com/AntennaPod/AntennaPod/blob/develop/config/checkstyle/checkstyle-new-code.xml) config file that can be imported in Android Studio.

Reply:
Thanks for the explanation. I wasn't aware of [diff-checkstyle](https://github.com/yangziwen/diff-checkstyle).